src/openrct2/paint/Supports.cpp

```diff
@@ -653,12 +706,18 @@ bool wooden_b_supports_paint_setup(
     {
         uint16_t specialIndex = (special - 1) & 0xFFFF;
 
-        uint32_t imageId = WoodenCurveSupportImageIds[supportType];
+        int32_t imageId;
+        if (specialIndex < NumVanillaWoodenSupportTypes)
+            imageId = WoodenCurveSupportImageIds[supportType];
```

Comment:
I don't like this exposing the g2 compared to normal elements. I'll submit a change that adjusts it

Reply:
@duncanspumpkin Can you explain what you want changed? I can't add sprites to g1.dat so that's why i have this awkward conditional; the game expects all the support sprites to be in a contiguous block.

Reply:
Yes so I want to change it so that the game doesn't expect it to be in a contiguous block